plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -432,7 +423,7 @@ public String getPluginVersion(File file) {
             Attributes attributes = manifest.getMainAttributes();
             return attributes.getValue("Plugin-Version");
         } catch (IOException e) {
-            e.printStackTrace();
+            LOGGER.log(Level.WARNING, "Unable to get plugin version from " + file.toString());
```

Comment:
```suggestion
            LOGGER.log(Level.WARNING, "Unable to get plugin version from {0}", file);
```

For better performance